pages/common/smalltalkci.md

```diff
@@ -0,0 +1,28 @@
+# smalltalkci
+
+> Framework for testing Smalltalk projects with GitHub Actions, Travis CI, AppVeyor, GitLab CI, and others.
+> More information: <https://github.com/hpi-swa/smalltalkCI#readme>
+
+- Run tests for a config file:
+
+`smalltalkci {{/path/to/.smalltalk.ston}}`
+
+- Run tests for the `.smalltalk.ston` configuration in the current directory:
+
+`smalltalkci`
+
+- Debug tests in headful mode (show VM window):

```

Comment:
`VM` is virtual machine, right?

Reply:
Correct. I think everyone who works with Smalltalk should have a basic understanding of the terms VM + image, so I would rather not explain this here in greater detail, would you agree with that?